### ISSUE TITLE
Alan Wake 2 Frame Generation wont enable

### ISSUE THREAD

**Issue by Gavinbots1** [Nukem9/dlssg-to-fsr3#82]
how were you able to get aw2 to work with this mod? its showing the popup but frame generation is still greyed out and won't even let me turn it on. I've tired all the versions and everything everybody said and have no hdr on. can somone help please rewlly want to use it for alen wake 2 
I have a 3080 and i7 12700f

**Comment by Ismael-Boff-UTN** [comment]
weird , last version 0.50 solved the problem for me, rtx 3060 btw, do u have gpu hw acelerator enabled from windows?

**Comment by kripikripi** [comment]
For me, it too shows the popup but the frame generation is gray

**Comment by norsende** [comment]
Same for me. Worked perfectly for Cyberpunk 2077, but Frame Generation remains greyed out in Alan Wake 2.

**Comment by RafaelNobre21** [comment]
i think it is running but you should not enable the frame gen cause for me still crash but im playing on ultra with rtx 2060 at 70 fps, 100 percent is running frame generation, my game used to run at  30 with ultra

**Comment by RafaelNobre21** [comment]
barely 30

**Comment by Nooblet16** [comment]
The dll method keeps the option disabled, the Tweaks method makes it available.
The only reason I know it works is because previously with 0.40 I had tried the Tweaks method and dll method, and only the tweaks method made the option clickable. So I left it at that and the game crashed.

This morning I put the 0.50 in using the dll method, the frame gen option was greyed out but ticked on from my previous run so it worked.

Basically 0.50 works using either methods provided it's already turned on, but only the Tweaks method will result in the option being actually clickable for toggle.

**Comment by norsende** [comment]
Hmm, I tried going ingame with Raytracing on High and Path Tracing enabled and it does seem to run at a way higher framerate which would mean that the Frame Generation is on, even though it is greyed out in the settings.

**Comment by Gavinbots1** [comment]
I do have the hardware excellent on and it works for cyberpunk but I've tied the tweaked one and still cant get the opinion to enable for me

**Comment by Gavinbots1** [comment]
yes I downloaded both the dlsstweaksa and the dlss to for tweaks version

**Comment by Gavinbots1** [comment]
could you send a zip of the files you used by chance? because I stayed up late night trying to figure it out but I gotta go to work rn so when I come home I can try again.

**Comment by Gavinbots1** [comment]
like the zip file that let's you enable the opinion for it please and thank you so much

**Comment by Styphoryte** [comment]
> like the zip file that let's you enable the opinion for it please and thank you so much

Dude if you took the time to look it's already been posted: (https://github.com/Nukem9/dlssg-to-fsr3/issues/75)
